pegasus/sites.v3/code.org/views/student_dance_projects.haml

```diff
@@ -2,41 +2,50 @@
   dance_projects = [
     {
       title: "Dance Party 1",
-      name: "Hadi",
+      name: "Ryan",
       age: "13+",
+      image: "ryan_project",
+      link: "https://studio.code.org/projects/dance/cMFiZ9Q2Nr2fDLsrzVRW0w"
     },
     {
-      title: "Dance Party 2",
-      name: "Hadi Again",
-      age: "13+",
+      title: "Mi oficina",
+      name: "Emma!",
+      age: "18+",
+      image: "emma_project",
+      link: "https://studio.code.org/projects/dance/TMhwhq8fQAeTEaoDS10ML-lhxiMGdKlHa6Lsvo1sTnA"
     },
     {
       title: "Dance Party 3",
       name: "Still Hadi",
       age: "13+",
+      image: "ryan_project",
+      link: ""
     },
     {
       title: "Dance Party 4",
       name: "Guess who",
       age: "13+",
+      image: "emma_project",
+      link: ""
     },
   ]
 
 .student_dance_projects
   - dance_projects.each do |project|
-    .project-card{style: "height: 240px; width: 216px; border: 1px solid gray; display: inline-block; margin: 10px;"}
-      .project-video
-        %img{src:"/images/animated-examples/cutesprite.gif", style: "height: 150px; width:100%"}
-      .project-details{style: "padding: 10px;"}
-        .project-title{style: "font-size: 18px; font-weight: bold;"}
-          = project[:title]
-        .project-author
-          .span{style: "font-size: 14px; font-weight: bold; display: inline-block;"}
-            By:
-          .span{style: "font-size: 14px; display: inline-block;"}
-            = project[:name]
-        .project-age
-          .span{style: "font-size: 14px; font-weight: bold; display: inline-block;"}
-            Age:
-          .span{style: "font-size: 14px; display: inline-block;"}
-            = project[:age]
+    %a{href: project[:link], style: "color: #696969;"}
```

Comment:
@madelynkasula once this goes in, I want to do the same thing as with the dance-teacher-resources and extract the styles to the css folder I made in that PR.